docs/specs/user-model.md

```diff
@@ -374,6 +375,27 @@ which OOB-OTP authenticator to use when performing authentication. However, a
 limit on the maximum amount of secondary OOB-OTP authenticators may be set in
 the configuration.
 
+##### Magic Link
+
+Magic link is a sub-category under OOB-OTP authenticator. User receives a login link instead of OTP code, which authenticates user on visit.
+
+Currently magic link is only available as secondary authenticator, and can be setup without verification. It will become verified once user successfully authenticates with magic link.
+
+```yaml
+authentication:
+  oob_otp:
+    email:
+      seconday_email_otp_mode: "magic_link"
```

Comment:
Not sure if we will want to allow a "email_otp_mode" option as well?

Reply:
No need for now.